### PR TITLE
decision records

### PR DESCRIPTION
**Pull request by priyanka-surana**
Add the decision records folder to `blobtoolkit`

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/blobtoolkit/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/blobtoolkit _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the unit test suite passes (`nextflow run . -profile test,singularity,sanger --outdir <OUTDIR>`) locally.
- [ ] Ensure the full test suite passes (`nextflow run . -profile test_full,singularity,sanger --outdir <OUTDIR>`) locally.
- [ ] Ensure that the CI tests pass by checking the nextflow tower link at the bottom of the CI tests and check the output on farm5 (the CI test only tells you if the job was successfully submitted to nextflow tower).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: docs/decision-records/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+Design decisions about the pipeline are indexed and recorded as individual files in this directory.
+
+To add a new decision, please create a pull request that adds a new markdown file named `XX-short-summary.md` to this directory. When replacing a previous decision, change the status of the latter to "Superseded". The new file should have the following structure:
```
Comment (zb32) [inline docs/decision-records/README.md:3]: To add a new decision, please create a pull request that adds a new markdown file named `XX-short-summary.md` to this directory. When replacing a previous decision, change the status of the latter to "Superseded" and add this to the title of the file `superseded-XX-short-summary.md`. The new file should have the following structure:

File: docs/decision-records/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+## Title – Decision Statement
+
+## Status – Either Proposed, Rejected, Current, Deprecated or Superseded
```
Comment (zb32) [inline docs/decision-records/README.md:7]: If 'Superseded' add a link to the new decision record

File: docs/decision-records/README.md (hunk 1 above)
Comment (zb32) [inline docs/decision-records/README.md:3]:
```suggestion
To add a new decision, please create a pull request that adds a new markdown file named `XX-short-summary.md` to this directory. When replacing a previous decision, change the status of the latter to "Superseded" and add this to the title of the file `superseded-XX-short-summary.md`. The new file should have the following structure:
```

File: docs/decision-records/README.md (hunk 2 above)
Comment (zb32) [inline docs/decision-records/README.md:7]:
```suggestion
## Status – Either Proposed, Rejected, Current, Deprecated or Superseded
If 'Superseded' add a line saying 'Superseded by <link-to-new-record>'
```

**Review by zb32 — COMMENTED** [review]
Just a few changes to make it clear when a decision is superseded and where the new decision record is :)

**Review by muffato — APPROVED** [review]

**Pull request merged**